api/v1alpha8/openstackcluster_webhook.go

```diff
@@ -63,6 +63,20 @@ func (r *OpenStackCluster) ValidateCreate() (admission.Warnings, error) {
 		allErrs = append(allErrs, field.Forbidden(field.NewPath("spec", "identityRef", "kind"), "must be a Secret"))
 	}
 
+	if r.Spec.ManagedSecurityGroups != nil && len(r.Spec.ManagedSecurityGroups.AllNodesSecurityGroupRules) > 0 {
```

Comment:
Does the len > 0 accomplish anything? :thinking:

Reply:
Not much indeed in this case.